modules/virt-configuring-cdiuploadproxy-routes.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+
+// * virt/post_installation_configuration/virt-post-install-network-config.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-configuring-cdiuploadproxy-routes_{context}"]
+= Configuring additional routes to the `cdi-uploadproxy` service
+
+[role="_abstract"]
+Configuring additional routes to the `cdi-uploadproxy` service enables external access for image uploads to virtual machine disks. Performing this configuration allows for the upload of virtual machine images from outside the cluster.
+
+.Prerequisites
+
+* You installed the OpenShift CLI (`oc`).
+* You logged in to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Configure the route to the external host:
```

Comment:
```suggestion
. Configure the route to the external host by running the following command:
```
See https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#procedure-step-that-introduces-a-command